system-apps/app-prop-viewer/pluginDefinition.json

```diff
@@ -15,7 +15,9 @@
     "descriptionDefault": "Zowe Viewer",
     "defaultWindowStyle": {
       "width": 350,
-      "height": 450
+      "height": 450,
+      "x": 0,
```

Comment:
I thought we didn't support x and y here anymore. I know other plugins have this, but it's ignored. Did it somehow get into the json schema? Why are we adding it to more places when we aren't using it?

Reply:
It's in the ticket for using json-schema, so I was adding it.

Reply:
We can leave it in here for now. However, we should remove the x, y coordinates from the json schema, as the fields are not currently used.

At best, x, y could be made optional.